### ISSUE TITLE
[bug]: Docker compose is broken

### ISSUE THREAD

**Issue by Igorgro** [makeplane/plane#997]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behavior

I started plane using instruction from readme (run `setup.sh localhost`, `docker compose up`, etc) from develop branch. Now requests from frontend to backend are broken. It tries to send request to non-existent `http://localhost1234` address. I found that this address is mentioned in frontend container startup logs:
```
planefrontend  | + /usr/local/bin/replace-env-vars.sh http://localhost:8000 http://localhost1234
planefrontend  | Replacing all statically built instances of http://localhost:8000 with this string http://localhost1234 .
```

I'm not sure but it looks like this line https://github.com/makeplane/plane/blob/develop/apps/app/Dockerfile.web#LL39C84-L39C106 is called, but .env file doesn't contains `NEXT_PUBLIC_WEBAPP_URL` variable

### Steps to reproduce

Delpoy Plane using instruction from Readme

### Browser

Google Chrome

### Version

Local

**Comment by pablohashescobar** [comment]
Hi @Igorgro,

We have pushed a fix for the above issue, we have updated our docker files and still testing the new setup. Let us know if you still find any issues.

**Comment by Igorgro** [comment]
@pablohashescobar  I performed `git pull`, cleared docker cache, rebuilt all images but still have exactly the same issue and the same lines in the frontend logs

**Comment by Igorgro** [comment]
Develop

**Comment by Igorgro** [comment]
Fixed by #1001, thanks
